src/product.html

```diff
@@ -15,12 +15,12 @@
         <!-- endbower -->
         <link rel="stylesheet" href="css/main.css">
     </head>
-    <body>
+    <body ng-app="data-fetch">
```

Comment:
thanks for the input. i havent edited this since Monday even you told me to, i just made this PR yesterday (Wed) to so that I will have something to remind me later... I plan to do all the corrections when time permits